### PR TITLE
[amd-osx.com] New ruleset.

### PR DESCRIPTION
**Pull request by Manouchehri**
CloudFlare hosted site.

### CODE REVIEW

File: src/chrome/content/rules/amd-osx.com.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<ruleset name="AMD OS X">
+	<target host="amd-osx.com" />
+	<target host="*.amd-osx.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/amd-osx.com.xml]: Please avoid wildcard targets if possible. You can simply add `forum.amd-osx.com` and `www.amd-osx.com.

File: src/chrome/content/rules/amd-osx.com.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+	<securecookie host="^([\w.-]+)\.amd-osx\.com$" name=".+" />
+
+	<test url="http://forum.amd-osx.com/" />
+	<test url="http://amd-osx.com/" />	
+	<test url="http://forum.amd-osx.com/search.php?search_id=active_topics" />
+	<test url="http://forum.amd-osx.com/memberlist.php?mode=contactadmin/" />
+
+	<rule from="^http://([\w.-]+)\.amd-osx\.com/"
+		to="https://$1.amd-osx.com/" />
```
Comment (J0WI) [inline src/chrome/content/rules/amd-osx.com.xml]: We prefer `<rule from="^http:" to="https:" />` for simple rules.

File: src/chrome/content/rules/amd-osx.com.xml (hunk 1 above)
Comment (Manouchehri) [reply src/chrome/content/rules/amd-osx.com.xml]: Fixed.

File: src/chrome/content/rules/amd-osx.com.xml (hunk 2 above)
Comment (Manouchehri) [reply src/chrome/content/rules/amd-osx.com.xml]: Thanks, fixed.

File: src/chrome/content/rules/amd-osx.com.xml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<ruleset name="AMD OS X">
+	<target host="amd-osx.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/amd-osx.com.xml]: Please also add `www.amd-osx.com` as target.

Comment (Manouchehri) [reply src/chrome/content/rules/amd-osx.com.xml]: Done.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**